test/integration/scheduler/scheduler_test.go

```diff
@@ -134,7 +134,7 @@ func TestSchedulerCreationFromConfigMap(t *testing.T) {
 				},
 				"ScorePlugin": {
 					{Name: "NodeResourcesBalancedAllocation", Weight: 1},
-					{Name: "PodTopologySpread", Weight: 1},
+					{Name: "PodTopologySpread", Weight: 2},
```

Comment:
I would not have expected a configuration change here

Reply:
Explained in https://github.com/kubernetes/kubernetes/pull/91598/files?file-filters%5B%5D=.go&file-filters%5B%5D=.json#r433494354